pkg/hints/math_hints.go

```diff
@@ -187,6 +187,54 @@ func sqrt(ids IdsManager, vm *VirtualMachine) error {
 	return nil
 }
 
+/*
+Implements hint:
+
+	%{
+	    from starkware.cairo.common.math_utils import assert_integer
+	    assert_integer(ids.div)
+	    assert 0 < ids.div <= PRIME // range_check_builtin.bound, \
+	        f'div={hex(ids.div)} is out of the valid range.'
+	    ids.q, ids.r = divmod(ids.value, ids.div)
+	%}
+*/
+func unsignedDivRem(ids IdsManager, vm *VirtualMachine) error {
+	div, err := ids.GetFelt("div", vm)
+	if err != nil {
+		return err
+	}
+	value, err := ids.GetFelt("value", vm)
+	if err != nil {
+		return err
+	}
+
+	rcBound, err := vm.GetRangeCheckBound()
+	if rcBound.Cmp(lambdaworks.FeltZero()) == 0 {
```

Comment:
```suggestion
	if rcBound.IsZero() {
```
Also, this case is not possible if we use a constant bound, so we can skip it

Reply:
You are right. I removed this check and added a test so that we can be sure the range check bound is never zero.